### PR TITLE
Note that the project is not compiling anymore

### PR DESCRIPTION
**Pull request by mheinzel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 A prototype MLS server. See https://github.com/wireapp/melissa.
 
+**Note: this project is currently not maintained and does not compile anymore.
```
Comment (jschaul) [inline README.md]: Does it still compile with stack version < 2 ? If so, change the note to reflect that?

**Comment by makingthematrix** [comment]
I'm also not able to compile it. To be honest, I'm thinking about rewriting it from scratch, but not in Haskell. Scala or Rust, more likely. But I would need some help in understanding what's going on here and why.

File: README.md (hunk 1 above)
Comment (mheinzel) [reply README.md]: Yes, it does. Fixed!

**Comment by mheinzel** [comment]
If you are interested in continuing development of this, we can find some time to make it work again. The problem is that the way we use symlinks in wire-server leads to issues with Stack, and it's a general problem (https://github.com/zinfra/backend-issues/issues/1216).

The advantage of having it in Haskell is that it makes it easy to re-use the prototype code once MLS is ready for production and we want to adopt it in wire-server.

If you want let's discuss the plans for this project. Maybe we can help somehow.

**Pull request merged**